hw2/05-form.js

```diff
@@ -0,0 +1,30 @@
+const form = document.querySelector('form');
+
+function handleSubmission(evnt) {
+  evnt.preventDefault(); // prevents screen from refreshing
+  const courseList = [];
+  const {
+    name, email, registration, courses, comment, fullStackInput, operatingInput, languagesInput,
+  } = form.elements;
```

Comment:
I really like this line and how you got all the elements. nice

Reply:
Thanks!